src/Illuminate/Database/Query/Builder.php

```diff
@@ -1436,6 +1436,19 @@ public function orderBy($column, $direction = 'asc')
     }
 
     /**
+     * Add a descending "order by" clause to the query.
+     *
+     * @param  string  $column
+     * @return $this
+     */
+    public function orderByDesc($column)
+    {
+        $this->orderBy($column, 'desc');
+
+        return $this;
```

Comment:
Here you go @CupOfTea696 https://github.com/laravel/framework/pull/18465

Thanks. 😄